app/gateway/logs.md

```diff
@@ -68,7 +68,69 @@ This feature can be customized for upstreams and downstreams using the `headers`
 
 ## Customize what {{site.base_gateway}} logs
 
-You might need to customize what {{site.base_gateway}} logs to protect private information and comply with GDPR. For example, if you wanted to remove instances of an email address from your logs. For more information about how to customize your logs, see [Customize Gateway Logs](/gateway/customize-gateway-logs/).
+You might need to customize what {{site.base_gateway}} logs to protect private information and comply with GDPR. For example, if you wanted to remove instances of an email address from your logs. These changes can be made to {{site.base_gateway}}'s Nginx template and only affect the output of the Nginx access logs. This doesn't have any effect on {{site.base_gateway}}'s [logging plugins](/plugins/?category=logging).
+
+For this example, let’s say you want to remove any instances of an email address from your {{site.base_gateway}} logs. The email addresses may come through in different ways, for example `/servicename/v2/verify/alice@example.com` or `/v3/verify?alice@example.com`. To keep all of these formats from being added to the logs, you need to use a custom Nginx template by making a copy of {{site.base_gateway}}'s Nginx template.
+
+The following template shows an example configuration to remove email addresses from logs, but you can use this logic to remove anything you want from the logs in a conditional manner:
```

Comment:
```suggestion
Make a copy of {{site.base_gateway}}'s Nginx template, then edit it to add or remove the data you need. The following template shows an example configuration for removing email addresses from logs:
```
I don't think you need the "you can use this same config for everything", that seems implied. Otherwise we'd put that in front of every example.

Reply:
We should link to wherever this info lives now, so they can figure out how to find the template: https://docs.konghq.com/gateway/latest/reference/nginx-directives/#custom-nginx-templates-and-embedding-kong-gateway